flask_saml_sso/sso.py

```diff
@@ -118,6 +115,31 @@ def wrapper():
     return wrapper
 
 
+@blueprint.route('/api_token/')
```

Comment:
`_` is somewhat unusual in paths:
```suggestion
@blueprint.route('/api-token/')
```